picturepoint/src/Layouts/Taskbar.jsx

```diff
@@ -0,0 +1,44 @@
+//React
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+
+//Material UI
+import { makeStyles } from '@material-ui/core/styles';
+import Paper from '@material-ui/core/Paper';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+
+//Style
+const useStyles = makeStyles({
+    root: {
+      flexGrow: 1,
+    },
+});
+
+//Render
+function Taskbar(props) {
+    const classes = useStyles();
+    const [value, setValue] = useState(0); //<- BUG here
```

Comment:
why is there a bug ? What needs to be fixed ?

Reply:
Currently, the taskbar is set to a default value of 0 which will set it to "My Friends". However, this creates a bug where refreshing the page always sets the taskbar back to "My Friends". I need to create a variable to keep track of the current page so refreshing doesn't change the taskbar.

Reply:
perfect ! thanks for the short demo on discord 💯